apps/seqan_flexbar/seqan_flexlib.cpp

```diff
@@ -257,11 +271,17 @@ void ArgumentParserBuilder::addAdapterTrimmingOptions(seqan::ArgumentParser & pa
         "np", "no-paired", "Trim paired-end input with single-end trimming method.");
     addOption(parser, pairedModeOpt);
 
-    seqan::ArgParseOption rateOpt = seqan::ArgParseOption(
+    seqan::ArgParseOption errorOpt = seqan::ArgParseOption(
```

Comment:
There Error-rate parameter that existed was not really a "rate" parameter, it should be called maximum number of allowed errors. However most papers specifiy an error rate, which is the -e option in the cutadapt tool. It is (maximum number of allowed errors)/overlap. In order not to break existing use cases of flexbar, i called the new option -er.